src/client/AM.PA.MacOs/PersonalAnalytics/Trackers/EmotionTracker/EmotionTrackerHelpers.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  EmotionTrackerHelpers.swift
+//  PersonalAnalytics
+//
+//  Created by Luigi Quaranta on 2019-01-15.
+//
+
+import Foundation
+
+struct EmotionTrackerHelpers {
+
+    // Properties
+    var context: NSManagedObjectContext?
+
+    // MARK: Initializer
+    init() {
+        // COREDATA INITIALIZATION
+        // Initialize a Persistent Container
+        let container = NSPersistentContainer(name: "PersonalAnalytics")
```

Comment:
Please put all interaction with the persistent store in the DataObjectController class. You shouldn't need to do any further initialization than what's already done there.